api/server.py

```diff
@@ -315,8 +353,14 @@ def healthcheck():
 
 
 @APP.route("/cr3/download/<crash_id>")
-@cross_origin(headers=["Content-Type", "Authorization"])
-@cross_origin(headers=["Access-Control-Allow-Origin", CORS_URL])
+@cross_origin(
+    headers=[
+        "Content-Type",
+        "Authorization",
+        "Access-Control-Allow-Origin",
+        CORS_URL,
+    ],
+)
```

Comment:
it seems like a best practice not to repeat the `cross_origin` decorator, but it's probably NBD 🤷